atom/browser/atom_permission_manager.cc

```diff
@@ -21,9 +21,7 @@ namespace {
 bool WebContentsDestroyed(int process_id) {
   auto contents =
       WebContentsPreferences::GetWebContentsFromProcessID(process_id);
-  if (!contents)
-    return true;
-  return contents->IsBeingDestroyed();
+  return (!contents) ? true : contents->IsBeingDestroyed();
```

Comment:
The old code was a clear early return, while the new code adds more complexity in my opinion.